typescript/src/electrum.ts

```diff
@@ -374,13 +374,25 @@ export class Client implements BitcoinClient {
           height: number
         }
 
-        const scriptHashHistory: HistoryEntry[] = await this.withBackoffRetrier<
-          HistoryEntry[]
-        >()(async () => {
-          return await electrum.blockchain_scripthash_getHistory(
-            scriptHash.reverse().toString("hex")
-          )
-        })
+        let scriptHashHistory: HistoryEntry[] = []
+        for (let attempt = 0; attempt < this.totalRetryAttempts; attempt++) {
```

Comment:
Not sure how would you like to see it, but please note that:
- there's a for loop in `withBackoffRetrier`, so if we wrap `withBackoffRetrier` with the loop that you marked then technically it can loop 9 times instead of 3.
- The idea was to have in total 3 attempts no matter if it returns the empty data or throws an error. If we put `withBackoffRetrier` outside of the for loop, then it will be more than 3 times.

Reply:
Do we need this change at all after we introduced a sleep at the beginning of the `waitTransactionConfirmed`?

This file `typescript/src/electrum.ts` is used not only in the system tests, it's a generic electrum integration.

Reply:
You know what.. you're right, the change in bitcoin.ts was actually enough. I run it locally 3 times and CI also has passed. [0cbfe8a](https://github.com/keep-network/tbtc-v2/pull/606/commits/0cbfe8a0484829c688eaf2c00c960466975ddc4c)